starter_code/app.js

```diff
@@ -9,9 +9,33 @@ const punkAPI = new PunkAPIWrapper();
 app.set('view engine', 'hbs');
 app.set('views', __dirname + '/views');
 app.use(express.static(path.join(__dirname, 'public')));
+app.set('laylout', __dirname + '/views/layout.hbs');
+hbs.registerPartials(__dirname + '/views/partials');
 
+//home page
 app.get('/', (req, res, next) => {
-  res.render('index');
+  res.render('index.hbs');
+});
+
+//beers page
+app.get('/beers', (req, res, next) => {
+   punkAPI.getBeers()
+   .then(beers => {
+    res.locals.beers = beers;
+    res.render('beers.hbs');
+    console.log(beers);
+    
+  })
+  
+  .catch(error => {
+    console.log(error)
+  })
+  
+});
+
+//random-beers page
+app.get('/random-beers', (req, res, next) => {
```

Comment:
Here you are missing the logic of calling the random beer method and passing the result as data to the view